primitive-types/impls/serde/src/serialize.rs

```diff
@@ -78,7 +78,7 @@ pub enum FromHexError {
 impl std::error::Error for FromHexError {}
 
 impl fmt::Display for FromHexError {
-	fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
+	fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
```

Comment:
seems redundant?

Reply:
this was generated by `cargo fix --edition-idioms` so I assumed it's part of the idioms lint, see also https://doc.rust-lang.org/edition-guide/rust-2018/ownership-and-lifetimes/the-anonymous-lifetime.html

but if you think this is useless, I can revert the change

Reply:
I don't see why we need useless generic argument, generated or not 🧐